chainguard-dev/edu
# Update rumble chart theme with light/dark colours

## Type of change
platform

### What should this PR do?
This updates the theme colours for Rumble vulnerability comparison charts depending on light/dark mode.

There's also a function to fetch wolfi advisory data mixed in here, but it is not used yet.

### Why are we making this change?
Visual consistency with the rest of our site theme.

### What are the acceptance criteria? 
The screenshots here show light and dark modes.
<img width="907" alt="Screenshot 2023-10-01 at 12 24 39 PM" src="https://github.com/chainguard-dev/edu/assets/328553/406b2167-77b7-4957-8a67-3bf0a1304857">
<img width="907" alt="Screenshot 2023-10-01 at 12 24 45 PM" src="https://github.com/chainguard-dev/edu/assets/328553/ea170fb1-b6fd-4775-8dc5-466d085e4f08">



### How should this PR be tested?
Testing locally should work, netlify won't. Toggling light/dark needs a page reload to get the chart to pick up on the change.
